src/core_plugins/kibana/public/management/sections/indices/index.html

```diff
@@ -13,7 +13,7 @@ <h5 data-test-subj="createIndexPatternParent">
         </a>
       </h5>
     </div>
-    <ul class="list-unstyled">
+    <ul class="list-unstyled" data-test-subj="index-pattern-list">
```

Comment:
Very very nitpicky suggestion: can we use camel case instead? e.g. `indexPatternList`?

Reply:
Done.